### DISCUSSION TITLE
When will windows be supported?

### DISCUSSION THREAD

**Discussion by IaMaAVerYRANdOmPerSoN** [Genesis-Embodied-AI/genesis-world#250] [Q&A]
Camera:
[Genesis] [14:24:55] [ERROR] Camera not yet supported on Windows but is supposed to. Can anyone help?

Interactive Viewer:
![image](https://github.com/user-attachments/assets/3befe059-2109-4a49-8d0f-f7382ec3f474)

**Comment by Cfomodz** [comment]


**Reply by duburcqa** [reply]
This is wrong. Windows is now fully supported and disabling viewer unwillingly is not recommended.
